draft-ietf-rtcweb-jsep.xml

```diff
@@ -556,6 +556,64 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
       </section>
 
+      <section anchor="sec.imageattr" title="Video Size Negotiation">
+        <t>Video size negotiation is the process through which a receiver can
+          use the "a=imageattr" SDP attribute <xref target="RFC6236"/> to indicate
+          what video frame sizes it is capable of receiving. A receiver may have
+          hard limits on what its video decoder can process, or it may wish to
+          constrain what it receives due to application preferences, e.g. a
+          specific size for the window in which the video will be displayed.
+        </t>
+
+        <section title="Creating an imageattr Attribute">
+          <t>In order to determine the limits on what video resolution a
+            receiver wants to receive, it will intersect its decoder hard
+            limits with any mandatory constraints that have been applied to
+            the associated MediaStreamTrack. The resulting resolution space
+            will be used as the x= and y= values for the generated "a=imageattr"
```

Comment:
yes, or really any time prior to SetLD(answer).

On the offerer side, this is problematic, as the track doesn't exist prior to setRD. While this could be addressed by a separate offer-answer exchange, this suggests that this mechanism is not ideal.

Agree the text should discuss this.